engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/CreateProcessInstanceRejectionTest.java

```diff
@@ -20,12 +20,14 @@
 import io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent;
 import io.camunda.zeebe.test.util.record.RecordingExporter;
 import io.camunda.zeebe.test.util.record.RecordingExporterTestWatcher;
+import io.camunda.zeebe.util.ByteValue;
 import org.assertj.core.api.Assertions;
 import org.junit.Rule;
 import org.junit.Test;
 
 public class CreateProcessInstanceRejectionTest {
 
+  public static final long MAX_MESSAGE_SIZE = ByteValue.ofMegabytes(4);
```

Comment:
:wrench: We could change the visibility of the constant.

```suggestion
  private static final long MAX_MESSAGE_SIZE = ByteValue.ofMegabytes(4);
```